theano/compile/mode.py

```diff
@@ -226,9 +226,10 @@ def apply(self, fgraph):
 optdb.register('add_destroy_handler', AddDestroyHandler(),
                49.5, 'fast_run', 'inplace')
 
+# 'merge3' seems useless
```

Comment:
Can you also run all Theano tests with profile=True to be sure there isn't corner case where this is not needed?

thanks